components/AddToCartConfirmationModal.vue

```diff
@@ -152,6 +152,7 @@ export default {
     const cartItemImage = computed(() => cartGetters.getItemImage(cartItem.value))
     const cartItemOptions = computed(() => cartGetters.getItemOptions(cartItem.value))
     const cartItemPrice = computed(() => cartGetters.getItemPrice(cartItem.value))
+    const getCartItemTotalPrice = computed(() => cartGetters.getAddToCartTotalPrice(cartItem.value))
```

Comment:
the leading "get" in variable name sounds like a function, use something like cartItemTotal

Reply:
This is removed , will be handled in a separate ticket.